packages/core/src/services/TextureLoader.ts

```diff
@@ -19,6 +19,7 @@ export class TextureLoader extends AbstractService {
         this.loader.setResponseType('blob');
         if (this.config.withCredentials) {
             this.loader.setWithCredentials(true);
+            this.loader.setCrossOrigin('use-credentials');
```

Comment:
FileLoader does not use this option https://github.com/mrdoob/three.js/blob/dev/src/loaders/FileLoader.js